src/app/components/MediaLoader/types.ts

```diff
@@ -46,6 +47,8 @@ export type PlaylistItem = {
   kind: string;
   duration: number;
   live?: boolean;
+  embedRights?: 'allowed';
+  vpid?: string;
```

Comment:
It appears that `vpid` is not a supported playlist setting, according to the SMP Cookbook, so I believe this could be deleted (and all references in tvMedia)